apps/src/netsim/netsim.js

```diff
@@ -508,6 +516,7 @@ NetSim.prototype.onRouterStateChange_ = function (router) {
   }
 
   this.setDnsMode(router.dnsMode);
+  this.setDnsNodeID(router.dnsMode === DnsMode.NONE ? undefined : router.dnsNodeID);
```

Comment:
I don't think so... are you referring to the level configuration?  Here, I legitimately want to say, "There is no DNS Node in `DnsMode.NONE`" and propagate this fact; in particular, so that the visualization knows to stop treating any node as the DNS node.  I could arguably build this logic into the visualization's `setDnsMode` handler though.